core/server/data/schema/fixtures/fixtures.json

```diff
@@ -1,5 +1,14 @@
 {
     "models": [
+        {
+            "name": "Product",
+            "entries": [
+                {
+                    "name":             "Default Product",
+                    "slug":             "default-product"
```

Comment:
Think we should get rid of the extra space here after `:` as we don't use it anywhere else in rest of the file.